### ISSUE TITLE
TaxCloud service not calculating tax

### ISSUE THREAD

**Issue by zenweasel** [reactioncommerce/reaction#3973]
## Issue Description
When enabled, the TaxCloud service is not calculating tax in the cart. There are no errors on the client or server console and I am using the current API key

## Steps to Reproduce
1. Enable TaxCloud tax service
1. Add an item to cart and enter address
1. Observe that tax is not calculated

### Versions
```
Node: 8.9.4
NPM: 5.6.0
Meteor Node: 8.9.4
Meteor NPM: 5.6.0
Reaction CLI: 0.28.0
Reaction: 1.9.0
Reaction branch: release-1.9.0
Docker: 17.12.0-ce
```

**Comment by zenweasel** [comment]
This is not a regression as it appears to be broken in `master`/`1.8.2` as well.

**Comment by zenweasel** [comment]
I tested back to 1.5.0 and it's not working there so not sure if this hasn't worked in a while or possibly I am misconfiguring something

**Comment by prinzdezibel** [comment]
Verified that this is not working

**Comment by zenweasel** [comment]
Let's keep this open till the code is merged

**Comment by prinzdezibel** [comment]
Assigning to @Akarshit , as he's working on that.

**Comment by spencern** [comment]
@Akarshit has this ticket been resolved by any of your recent PRs?

**Comment by Akarshit** [comment]
@spencern yes by #4140

**Comment by zenweasel** [comment]
Closed via #4140
